### ISSUE TITLE
Cop to check for blank lines at the beginning/end of a method

### ISSUE THREAD

**Issue by daviddavis** [rubocop/rubocop#1378]
Would be nice to have a cop that checks for this:

``` ruby

# bad
def test

  a + b
end

# bad
def test
  a + b

end

# good
def test
  a + b
end
```

I'm on 0.24.1 so apologies if this was added already.
